### DISCUSSION TITLE
How do I use this thing?

### DISCUSSION THREAD

**Discussion by NoisyChain** [nykwil/UnityGGPO#13] [Q&A] [answered]
A Unity port of GGPO is wonderful... But I wish I had some tutoarials of sort to make an actual game using it.

Where do I start to learn how to use this? (besides learning ECS of course)?

**Comment by nykwil** [comment, marked as answer]
I removed the ecs stuff. I would start with the vectorwar scene, there is a description of this sort of stuff in the readme if you have specific feedback.

**Reply by NoisyChain** [reply]
I already figured it out. I'm porting my fighting game project to UnityGGPO right now

Thank you so much for your work, you really saved my skin with this
